x-pack/plugins/security_solution/server/endpoint/services/agent/clients/lib/base_agent_status_client.ts

```diff
@@ -17,6 +18,7 @@ export interface AgentStatusClientOptions {
   endpointService: EndpointAppContextService;
   esClient: ElasticsearchClient;
   soClient: SavedObjectsClientContract;
+  connectorActionsClient: ActionsClient;
```

Comment:
```suggestion
  connectorActionsClient: ActionsClient | undefined;
```

or 

```suggestion
  connectorActionsClient?: ActionsClient;
```

Reply:
I think it's required for SentinelOne https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/endpoint/routes/agent/agent_status_handler.ts#L91

Reply:
The code you're pointing to does not use this type. That is the old sentinel one agent status method within `x-pack/plugins/security_solution/server/endpoint/services/agent/agent_status.ts`. This type is used by the new agent status client and starting with your PR `connectorActionsClient` is required only for `crowdstrike`